components/Carbon.js

```diff
@@ -149,9 +149,7 @@ class Carbon extends React.Component {
               border-left: solid 2px #159588;
             }
 
-            #container :global(.cm-s-solarized) {
-              background-color: #fdf6e3;
-              color: #657b83;
+            #container :global(.cm-s-solarized.cm-s-light) {
               text-shadow: #eee8d5 0 1px;
               box-shadow: none;
```

Comment:
You'll want to apply `box-shadow: none;` to solarized dark as well, it'll remove that weird shadow on the left edge of the terminal (this is seen in your second screenshot).